cmd/public-api-demo/main.go

```diff
@@ -43,6 +45,26 @@ func runDemo(identifier, version, arch, regcode string) error {
 		creds.ShowTraces = true
 	}
 
+	if certificatePath := os.Getenv("API_CERT"); certificatePath != "" {
+		crt, certReadErr := os.ReadFile(certificatePath)
+		if certReadErr != nil {
+			return certReadErr
+		}
+
+		block, _ := pem.Decode(crt)
+		if block == nil {
```

Comment:
shouldn't `err`  be part of the check?

Reply:
this is not a `err` its the `rest` of the decoded pem. I was first testing against this error... https://pkg.go.dev/encoding/pem#Decode